tools/crm_mon.c

```diff
@@ -648,15 +648,22 @@ static GOptionEntry deprecated_entries[] = {
 static gboolean
 reconnect_after_timeout(gpointer data)
 {
+    int rc = pcmk_rc_ok;
+    const char *desc = NULL;
 #if CURSES_ENABLED
     if (output_format == mon_output_console) {
         clear();
         refresh();
     }
 #endif
 
+    rc = pacemakerd_status(&desc);
+    if (desc != NULL) {
+        out->info(out, "%s", desc);
+    }
     out->info(out, "Reconnecting...");
```

Comment:
Good point -- although it will show "Cluster is not available" (and for console mode, "Connection to the cluster lost") before this. I think I grabbed the status first because in the third commit we do `"Reconnecting (<pcmkd_status>)"`. I'll rethink that